README.md

```diff
@@ -68,51 +80,50 @@ Clone this repository to your desired folder:
 Example commands:
 
 ```sh
- $ cd your-folder
- $ git clone https://github.com/mutugi-c/Webpack-boilerplate.git
+ $ cd my-folder
+ $ git clone git@github.com:mutugi-c/To_do_list.git
 ```
 
 ### Install
 
-To work with this boilerplate, you need to initialize npm, install webpack locally, and install the webpack-cli (the tool used to run webpack on the command line):
+Install git to work on this project with:
 
 ```sh
-   $ npm init -y
+   $ git config --global user.name "First Last"
 
-   $ npm install webpack webpack-cli --save-dev
+   $ git config --global user.email "example@example.com"
 ```
 
 ### Usage
 
-To build during development or for production, run:
-
-```sh
-  $ npm run build
-```
-
-otherwise, you can use the local dev server using:
+To run the project have Node.js installed, execute the following command:
 
 ```sh
-  $ npm start
+  $ npm install http-server -g
+  $ http-server
```

Comment:
- [ ] @mutugi-c great work on your readme, however, I would recommend you change the instructions to run this project, since this is a webpack project, I expected instructions eg `npm i` to install dependencies for the project, `npm start ` to run the project, same case to the running tests